File: GUI/Custom Controls/PictureBoxDoubleBuffer.vb
Imports System.Windows.Forms

Public Class PictureBoxDoubleBuffer
    Inherits PictureBox
    Public Sub New()
        Me.DoubleBuffered = True
        SetStyle(ControlStyles.AllPaintingInWmPaint Or ControlStyles.OptimizedDoubleBuffer Or ControlStyles.ResizeRedraw Or ControlStyles.UserPaint, True)
        Me.UpdateStyles()
    End Sub
End Class

File: Release Candidate/Platform/UI Layouts/Generic/Custom Controls/Custom Controls/PictureBoxDoubleBuffer.cs
using System.Windows.Forms;

public class PictureBoxDoubleBuffer : PictureBox
{
    public PictureBoxDoubleBuffer()
    {
        SuspendLayout();
        SetStyle(ControlStyles.AllPaintingInWmPaint, true);
        SetStyle(ControlStyles.OptimizedDoubleBuffer, true);
        // 'SetStyle(ControlStyles.SupportsTransparentBackColor, True)
        SetStyle(ControlStyles.ResizeRedraw, true);
        SetStyle(ControlStyles.UserPaint, true);
        UpdateStyles();
        ResumeLayout();
    }
}